src/commands.rs

```diff
@@ -354,6 +354,36 @@ impl AssetCommand {
         patch(&self.authentication, &endpoint, &json!({ "headers": map }))
     }
 
+    pub fn update_web_asset_headers(
+        &self,
+        id: &str,
+        headers: Vec<(String, String)>,
+    ) -> anyhow::Result<(), CommandError> {
+        // if no headers provided update has nothing to do
+        if headers.is_empty() {
+            return Ok(());
+        }
+
+        let mut new_headers: HashMap<_, _> = headers.into_iter().collect();
+        let asset = self.get(id)?;
```

Comment:
You can request headers only to reduce payload for get - /v4/assets?select=headers

Reply:
But it's the only place only headers are needed, and a single asset I don't see the reason to complicate it

Reply:
got it